flink-java/src/main/java/org/apache/flink/api/java/ClosureCleaner.java

```diff
@@ -101,6 +132,29 @@ public static void clean(Object func, boolean checkSerializable) {
 		}
 	}
 
+	public static void clean(Object func, boolean checkSerializable) {
```

Comment:
Do we strictly need the old `clean()` methods that has the default of "recursion". It might be more straightforward in the code to always require the config.

Reply:
I meant to keep the old `clean()` to avoid changing too much clean method, I can remove this.